dataGen.js

```diff
@@ -0,0 +1,1204 @@
+const song = {
```

Comment:
Ideally, since Ringo's function already generated data and we want each person to have the same general song data (for instance, we want everyone's song#1 to be Rick Astley), please use the array of data Ringo generated, import that array into this file, then use the '_.extends' function to add your desired properties in to the the existing song objects

Reply:
I wrote my own function to modify existing data, added in upcoming commit.